ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-master.yaml

```diff
@@ -68,6 +68,14 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-upgrade-gcp-loki
+- as: e2e-rollback
+  steps:
+    cluster_profile: gcp
+    env:
+      TEST_COMMAND: run-upgrade
```

Comment:
I don't think we need to set this explicitly, since it's the default:

```console
$ git --no-pager grep TEST_COMMAND ci-operator/step-registry/openshift/upgrade/gcp/loki/
ci-operator/step-registry/openshift/upgrade/gcp/loki/openshift-upgrade-gcp-loki-workflow.yaml:      TEST_COMMAND: "run-upgrade"
```

Reply:
`env` section completely overwrites existing one, so all parameters have to duplicated here.

I filed https://issues.redhat.com/browse/DPTP-1644 to investigate templating in step workflows